PolSourceTasks/src/classes/EditionTriggerHandler.cls

```diff
@@ -37,4 +29,12 @@ public with sharing class EditionTriggerHandler {
         }        
         updateBooks(bookIds);
     }
+    
+    private static void updateBooks(Set<Id> bookIds) {
```

Comment:
Jak będziesz miał w tym triggerze 15 różnych typów zmian i każda będzie update'owala Book'a to zaczniesz je numerować? ;p
Nazwij to tak, żeby było wiadomo co robi.

Nazewnictwo rzeczy jest trudniejsze niż się wydaje, czego studia oczywiście nie uświadamiają ;)